src/pages/default/credentials/Credentials.tsx

```diff
@@ -9,7 +9,7 @@ import { FlexLayouts } from "@/theme/layouts";
 import { BadgeNames, ItemProps } from "@/components/Item";
 
 const CredentialsScreen = ({ navigation }) => {
-  const navigateToItem = (credential) => {
+  const navigateToItem = (credential: ItemProps) => {
```

Comment:
There's still a little bit of work to be done to decouple this ItemProps from the various data model interfaces